.bumpversion.cfg

```diff
@@ -1,7 +1,7 @@
 [bumpversion]
 commit = False
 tag = False
-current_version = 3.0.0
+current_version = 3.1.0
```

Comment:
I think this should also be 3.0.1 since there are no new features

Reply:
Changed 3.1.0 to 3.0.1 version